cider/tests/functionality/CiderFilterTest.cpp

```diff
@@ -131,10 +131,9 @@ TEST_F(CiderFilterSequenceTestBase, inTest) {
   assertQuery("SELECT * FROM test WHERE col_3 not in (24, 25, 26)",
               "not_in_fp32_array.json");
   // TODO: add in (str_1, str_2, str_3)
```

Comment:
remove?

Reply:
noop, keep it and will support `in(string1, string2...)` later.

Reply:
Please add owner.